README.md

```diff
@@ -52,27 +57,65 @@ fplll is distributed under the [GNU Lesser General Public License](COPYING) (eit
 
 # Compilation #
 
-## Dependencies ##
+## Installation from packages ##
+
+fplll is available as a pre-built package for a variety of operating systems; these pre-built packages typically include all mandatory dependencies, and so these packages can be used to start running fplll quickly.
+
+Below, we give some instructions on how to install these packaged variants of fplll.
+
+Note that these packages are likely to be rebuilt every time a new tag is cut; as a result, if you want fully up-to-date code, then it is necessary to build from source (See the "Install from Source" section).
+
+### Ubuntu and Debian ###
+fplll can be installed directly via Aptitude or Synaptic. Both of these package managers package fplll in the package `fplll-tools`. Therefore, to install this package using Aptitude, run the following command
+
+``` 
+sudo aptitude install fplll-tools
+```
+
+If you want to use Synaptic, then you will need to search for the `fplll-tools` package using the search bar.
+
+
+### Conda ###
+fplll can be installed natively as a conda package using the following command
 
-### Required ###
+```
+conda install fplll
+```
+
+### MacOS ###
+
+MacOS has a package for fplll inside HomeBrew. Assuming that you have HomeBrew installed, you may install fplll using the following command
+
+```
+brew install fplll
+```
+
+### Docker and AWS ###
+We now have Docker/AWS images for fplll too. They aren't on this repository, though; you can find them [here](https://github.com/malb/fplll-virtual-machines)
```

Comment:
I'm in the process of changing that up better to link to https://hub.docker.com/u/fplll